### ISSUE TITLE
xStudio

### ISSUE THREAD

**Issue by aldmbmtl** [juno-fx/Terra-Official-Plugins#97]
I have taken a stab at this before, but that was while trying to get it to compile over a year ago now. This may be solvable now with Terra and building/compiling it so we can deploy it may be trivial. 

https://github.com/AcademySoftwareFoundation/xstudio/issues/36

Good luck...

**Comment by ddesmond** [comment]
After romaing trough the main repo and all updated forks i coud find, the conclusion is:
this is not compiling becouse code and styling issues in C codebase, mismatch in cmake version dependencies which prevent suscesfull builds.

I found two regular contributors forks and both had some patches but nothing is complete.
Tried to build vcpk aswell but they are missing libs and some libs they use are not available on linux.

Devbox works for fullfiling cmake version and python version and reqs but has many missing libs that we would have to write all deps for nix itself.

If we cant build essentials libs in nix store we cant use this for VFX apps which have deps in fron redhat world.

It seem this is a no go from both ways.
